### ISSUE TITLE
Show sucuri scan error

### ISSUE THREAD

**Issue by abdul891** [elementor/elementor#21989]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

When I scan url(**https://terceroinc.com/**) in sucuri then sucriri showing this issue

please see screenshot below
![Screenshot (1771)](https://user-images.githubusercontent.com/78399312/230903429-fd0e1e42-4668-47e9-bf8e-81aedc6d717f.png)

This issue appear from elementor header .when i remove header then sucuri not showing this error.

Please check this and how to fix it.

sucuri scan url https://sitecheck.sucuri.net/results/https/staging.terceroinc.com

### Steps to reproduce

I can not explain.

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: Percona Server (GPL), Release 23, Revision 500fcf5 v5.7.23-23
	PHP Version: 8.1.17
	PHP Memory Limit: 512M
	PHP Max Input Vars: 1000
	PHP Max Post Size: 516M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.1.1
	Site URL: https://terceroinc.com
	Home URL: https://terceroinc.com
	WP Multisite: No
	Max Upload Size: 512 MB
	Memory limit: 40M
	Max Memory limit: 512M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: 0
	Admin Email: 
	Debug Mode: Inactive

== Theme ==
	Name: TwentyTwentyOneChild
	Version: 1.0
	Author: Dharmendra
	Child Theme: Yes
	Parent Theme Name: Twenty Twenty-One
	Parent Theme Version: 1.7
	Parent Theme Author: the WordPress team

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/111.0.0.0 Safari/537.36

== Active Plugins ==
	Akismet Anti-Spam
		Version: 5.1
		Author: Automattic

	All in One SEO
		Version: 4.3.3
		Author: All in One SEO Team

	Classic Editor
		Version: 1.6.2
		Author: WordPress Contributors

	Contact Form 7
		Version: 5.7.5.1
		Author: Takayuki Miyoshi

	Contact Form CFDB7
		Version: 1.2.6.5
		Author: Arshid

	CookieYes | GDPR Cookie Consent
		Version: 3.0.8
		Author: CookieYes

	Elementor
		Version: 3.12.1
		Author: Elementor.com

	Elementor Pro
		Version: 3.12.1
		Author: Elementor.com

	Essential Addons for Elementor
		Version: 5.6.3
		Author: WPDeveloper

	Imagify
		Version: 2.1.1
		Author: Imagify – Optimize Images & Convert WebP

	Permalink Manager Lite
		Version: 2.3.1.1
		Author: Maciej Bis

	Redirection
		Version: 5.3.9
		Author: John Godley

	Redirection for Contact Form 7
		Version: 2.8.0
		Author: Qube One

	Strong Testimonials
		Version: 3.1.1
		Author: WPChill

	Wordfence Security
		Version: 7.9.2
		Author: Wordfence

	WP Activity Log
		Version: 4.4.3.2
		Author: WP White Security

	WP Rocket
		Version: 3.12.6.1
		Author: WP Media

	WPS Hide Login
		Version: 1.9.8
		Author: WPServeur, NicolasKulka, wpformation

== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	

== Elementor Experiments ==
	Optimized DOM Output: Active
	Improved Asset Loading: Active
	Improved CSS Loading: Active
	Inline Font Icons: Active
	Accessibility Improvements: Active
	Additional Custom Breakpoints: Active
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Active
	Upgrade Swiper Library: Inactive by default
	Default to New Theme Builder: Active by default
	Editor Top Bar: Inactive by default
	Landing Pages: Inactive
	Nested Elements: Inactive by default
	Lazy Load Background Images: Inactive by default
	Page Transitions: Active by default
	Notes: Inactive
	Loop: Active by default
	Form Submissions: Inactive
	Scroll Snap: Inactive
	Menu: Inactive by default

== Log ==
	
Log: showing 20 of 222023-02-24 11:10:56 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.10.2',
  'to' => '3.11.2',
)]
2023-03-27 07:11:11 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.2',
  'to' => '3.11.5',
)]
2023-03-27 07:11:13 [info] elementor::elementor_updater Started 
2023-03-27 07:11:13 [info] Elementor/Upgrades - _on_each_version Start  
2023-03-27 07:11:13 [info] Elementor/Upgrades - _on_each_version Finished 
2023-03-27 07:11:13 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.2',
  'to' => '3.11.5',
)]
2023-03-27 07:28:52 [info] elementor-pro::elementor_pro_updater Started 
2023-03-27 07:28:52 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-03-27 07:28:52 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-03-27 07:28:52 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.2',
  'to' => '3.11.7',
)]
2023-04-05 04:46:19 [info] elementor::elementor_updater Started 
2023-04-05 04:46:19 [info] Elementor/Upgrades - _on_each_version Start  
2023-04-05 04:46:19 [info] Elementor/Upgrades - _on_each_version Finished 
2023-04-05 04:46:19 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.5',
  'to' => '3.12.1',
)]
2023-04-05 04:46:19 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.11.5',
  'to' => '3.12.1',
)]
2023-04-05 04:46:23 [info] elementor-pro::elementor_pro_updater Started 
2023-04-05 04:46:23 [info] Elementor Pro/Upgrades - _on_each_version Start  
2023-04-05 04:46:23 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2023-04-05 04:46:23 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.7',
  'to' => '3.12.1',
)]
2023-04-05 04:46:23 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.11.7',
  'to' => '3.12.1',
)]

PHP: showing 8 of 8PHP: 2023-02-24 11:11:30 [notice X 668][/home3/terceinc/public_html/wp-content/plugins/elementor-pro/modules/custom-css/module.php::76] trim(): Passing null to parameter #1 ($string) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-02-24 11:19:17 [notice X 4][/home3/terceinc/public_html/wp-content/plugins/elementor/includes/template-library/sources/local.php::747] ob_end_clean(): Failed to delete buffer. No buffer to delete [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-02-24 11:19:23 [error X 3][/home3/terceinc/public_html/wp-content/plugins/elementor/core/documents-manager.php::476] Uncaught Error: Call to a member function is_built_with_elementor() on bool in /home3/terceinc/public_html/wp-content/plugins/elementor/core/documents-manager.php:476
Stack trace:
#0 [internal function]: Elementor\Core\Documents_Manager->ajax_save(Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#1 /home3/terceinc/public_html/wp-content/plugins/elementor/core/common/modules/ajax/module.php(175): call_user_func(Array, Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#2 /home3/terceinc/public_html/wp-includes/class-wp-hook.php(308): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request('')
#3 /home3/terceinc/public_html/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters('', Array)
#4 /home3/terceinc/public_html/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#5 /home3/terceinc/public_html/wp-admin/admin-ajax.php(188): do_action('wp_ajax_element...')
#6 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-02-24 11:41:44 [warning X 40][/home3/terceinc/public_html/wp-content/plugins/elementor/core/common/modules/ajax/module.php::175] Undefined array key "data" [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-02-25 05:35:46 [notice X 67][/home3/terceinc/public_html/wp-content/plugins/elementor-pro/core/database/model-base.php::135] Return type of ElementorPro\Core\Database\Model_Base::jsonSerialize() should either be compatible with JsonSerializable::jsonSerialize(): mixed, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-03-29 05:53:58 [error X 1][/home3/terceinc/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php::295] Uncaught Error: Call to a member function update_meta() on bool in /home3/terceinc/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php:295
Stack trace:
#0 /home3/terceinc/public_html/wp-content/plugins/elementor-pro/modules/theme-builder/classes/conditions-manager.php(168): ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->save_conditions(NULL, Array)
#1 [internal function]: ElementorPro\Modules\ThemeBuilder\Classes\Conditions_Manager->ajax_save_theme_template_conditions(Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#2 /home3/terceinc/public_html/wp-content/plugins/elementor/core/common/modules/ajax/module.php(175): call_user_func(Array, Array, Object(Elementor\Core\Common\Modules\Ajax\Module))
#3 /home3/terceinc/public_html/wp-includes/class-wp-hook.php(308): Elementor\Core\Common\Modules\Ajax\Module->handle_ajax_request('')
#4 /home3/terceinc/public_html/wp-includes/class-wp-hook.php(332): WP_Hook->apply_filters('', Array)
#5 /home3/terceinc/public_html/wp-includes/plugin.php(517): WP_Hook->do_action(Array)
#6 /home3/terceinc/public_html/wp-admin/admin-ajax.php(188): do_action('wp_ajax_element...')
#7 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-04-03 11:23:53 [warning X 19][/home3/terceinc/public_html/wp-content/plugins/elementor/app/modules/import-export/module.php::629] Attempt to read property "label" on null [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2023-04-05 04:46:25 [notice X 11502][/home3/terceinc/public_html/wp-content/plugins/elementor-pro/modules/loop-builder/module.php::204] strpos(): Passing null to parameter #1 ($haystack) of type string is deprecated [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

JS: showing 12 of 12JS: 2023-02-24 11:18:03 [error X 3][https://terceroinc.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.11.2:3:1087291] Cannot read properties of null (reading 'getBoundingClientRect') 
JS: 2023-02-24 12:51:07 [error X 4][https://terceroinc.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.11.2:3:875609] Cannot read properties of undefined (reading 'val') 
JS: 2023-02-24 13:47:13 [error X 4][https://terceroinc.com/wp-content/plugins/elementor/assets/js/common-modules.min.js?ver=3.11.2:2:53736] Model.isValidChild() should be implemented, please provide 'isValidChild' functionality. 
JS: 2023-02-24 15:35:15 [error X 7][https://terceroinc.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.11.2:3:791668] Cannot read properties of undefined (reading 'localized') 
JS: 2023-02-26 05:46:41 [error X 41][https://terceroinc.com/wp-admin/load-scripts.php?c=0&loadchunk_0=underscore,jquery-core,jquery-migrate,backbone,jquery-ui-core,jquery-ui-mouse,wp-api-request,regenerator-runtime,wp-polyfill,wp-&loadchunk_1=hooks&ver=6.1.1:4:51824] Cannot read properties of undefined (reading 'replace') 
JS: 2023-02-27 11:10:38 [error X 1][https://terceroinc.com/wp-admin/post.php?post=621&action=elementor&active-document=6#:7:51] Cannot read properties of undefined (reading 'tabs') 
JS: 2023-02-27 11:10:40 [error X 1][https://terceroinc.com/wp-admin/post.php?post=621&action=elementor&active-document=6#:6:39] Cannot read properties of undefined (reading 'needHelpUrl') 
JS: 2023-03-21 12:03:56 [error X 1][https://terceroinc.com/wp-content/plugins/elementor/assets/js/editor.min.js?ver=3.11.2:3:670939] this.model.isValidChild is not a function 
JS: 2023-03-22 04:09:18 [error X 4][https://terceroinc.com/wp-content/plugins/elementor-pro/assets/js/editor.min.js?ver=3.11.2:2:61404] Cannot convert undefined or null to object 
JS: 2023-03-28 07:16:18 [error X 11][https://terceroinc.com/wp-content/plugins/elementor-pro/assets/js/frontend.min.js?ver=3.11.7:2:20176] this.$element.sticky is not a function 
JS: 2023-03-30 08:56:47 [error X 2][https://terceroinc.com/wp-admin/load-scripts.php?c=0&loadchunk_0=underscore,jquery-core,jquery-migrate,backbone,jquery-ui-core,jquery-ui-mouse,wp-api-request,regenerator-runtime,wp-polyfill,wp-&loadchunk_1=hooks&ver=6.1.1:4:51816] a is undefined 
JS: 2023-03-30 17:40:36 [error X 5][https://terceroinc.com/wp-content/plugins/elementor/assets/lib/pickr/pickr.min.js?ver=1.5.0:2:19643] this.getColorRepresentation() is null 

== Elementor - Compatibility Tag ==
	
	Elementor Pro: Incompatible
	Essential Addons for Elementor: Incompatible

== Elementor Pro - Compatibility Tag ==
	
	Essential Addons for Elementor: Incompatible

**Comment by abdul891** [comment]
Any update ?
